view/frontend/web/js/view/payment/method-renderer/adyen-hpp-method.js

```diff
@@ -212,26 +212,57 @@ define(
                         var country = '';
                         var postalCode = '';
                         var street = '';
+                        var firstName = '';
+                        var lastName = '';
+                        var telephone = '';
+                        var email = '';
+                        var shopperGender = '';
+                        var shopperDateOfBirth = '';
 
                         if (!!quote && !!quote.shippingAddress()) {
-                             city = quote.shippingAddress().city;
-                             country = quote.shippingAddress().countryId;
-                             postalCode = quote.shippingAddress().postcode;
-                             street = quote.shippingAddress().street.join(" ");
+                            city = quote.shippingAddress().city;
+                            country = quote.shippingAddress().countryId;
+                            postalCode = quote.shippingAddress().postcode;
+                            street = quote.shippingAddress().street.join(" ");
+                            firstName = quote.shippingAddress().firstname;
+                            lastName = quote.shippingAddress().lastname;
+                            telephone = quote.shippingAddress().telephone;
+                            email = customerData.email;
+                            shopperGender = customerData.gender;
+                            shopperDateOfBirth = customerData.dob;
```

Comment:
have you tested it with guest user?

Reply:
The parameters are passing correct from the frontend to backend with the guest user